### ISSUE TITLE
two lines take no effect

### ISSUE THREAD

**Issue by penglei0** [chosen-ox/Bandit_UCB#11]
https://github.com/chosen-ox/Bandit_UCB/blob/4a94839734e64fa126944aae9c7a69dfc2e28225/rc80211_minstrel_ht.c#L1454

@chosen-ox @YzTongNiar 

Sorry to disturb you. Were those lines of code the normal implementation ?

**Comment by chosen-ox** [comment]
You are right!  It takes no effect but to pass the compilation. You can do some changes and if it can pass the compilation. You can start a PR and we will consider merging it! Thank you for your interest.
